catalog/security/catalog-security-logging/pom.xml

```diff
@@ -35,6 +35,10 @@
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-api-impl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.platform.util</groupId>
```

Comment:
Do we need this? I don't see it being used or any of it's dependencies needing it